packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx

```diff
@@ -90,6 +126,9 @@ export const ChatMessageActionFlyout = (props: ChatMessageActionFlyoutProps): JS
             props.increaseFlyoutItemSize ? menuItemIncreasedSizeStyles : undefined
           )
         },
+        subMenuProps: {
+          items: messageReadByList
```

Comment:
`messageReadByList` is inside a useMemo but I don't see it added to the dependency array -- should it? (there should be an eslint warning if it should be)